lib/rules/no-unused-class.js

```diff
@@ -107,14 +109,14 @@ export default {
            ```
            then the loop will be run 2 times
          */
-        _.forOwn(map, (o) => {
+        _.forIn(map, (o) => {
           const { classes, node } = o;
 
           /*
              if option is passed to mark a class as used, example:
              eslint css-modules/no-unused-class: [2, { markAsUsed: ['container'] }]
            */
-          _.forEach(markAsUsed, usedClass => {
+          markAsUsed && markAsUsed.forEach((usedClass) => {
```

Comment:
I had used `_.forEach` instead of forEach so that there would be no need for null checks like this.